ecoscope/analysis/seasons.py

```diff
@@ -33,11 +33,14 @@ def std_ndvi_vals(aoi=None, img_coll=None, band=None, img_scale=1, start=None, e
     else:
         geo = None
 
+    img_dates = pandas.to_datetime(coll.aggregate_array("system:time_start").getInfo(), unit="ms", utc=True)
+
+    coll = coll.map(lambda x: x.normalizedDifference(band))
```

Comment:
this needs to be two bands, NIR and Red so should be an array.

Reply:
It can be an array, though the param should be updated to `bands` to reflect this

Reply:
Ok, cool. The order matters so should probably have a comment to the user on this. For NDVI it will be [NIR, RED] in that order:
https://developers.google.com/earth-engine/apidocs/ee-image-normalizeddifference

Reply:
in that case, rather than `bands=[]` why not have `nir_band` and `red_band`?

Reply:
Yep, that's more explicit.